interface/modules/custom_modules/oe-module-faxsms/info.txt

```diff
@@ -0,0 +1 @@
+FaxSMS Module v3.0.0 for v7.0.1
```

Comment:
If this is for OpenEMR 7.0.1 I wonder if you should update the composer.json conflict to be for that version of OpenEMR.

Reply:
composer is not used and only reason I leave is to maybe make a stand alone install.